### DISCUSSION TITLE
How to embed tweets in blogpost?

### DISCUSSION THREAD

**Discussion by kurianbenoy** [quarto-dev/quarto-cli#4884] [Q&A] [answered]
Is there any official support of embedding tweets in blogpost?

I recently found out there is an extension supported by someone : https://github.com/sellorm/quarto-social-embeds. 

I am looking for an official support

**Comment by mcanouil** [comment, marked as answer]
If by "official" you mean embedded in Quarto, there is not.
Either use the extension or use plain HTML code from Twitter and put that code into your document.

**Comment by cderv** [comment]
> I am looking for an official support

Curious why the extensions is not a good fit to you ? Is it not working properly ? 
Extensions are official part of the ecosytem to add such features.

**Reply by kurianbenoy** [reply]
Actually when I was trying first, it was not really working when publishing while it worked in my local. Then I figured it out only later as I didn't put my `_extensions` folder in github.

If there is an official support, then it's easier to find and use it in quarto. That's why I checked for an official support.

**Reply by cderv** [reply]
Good to know it works. 

> If there is an official support, then it's easier to find and use it in quarto.

We maintain a list of known extensions to make them easier to find and use 
https://quarto.org/docs/extensions/listing-filters.html

In case you haven't find it before

**Reply by kurianbenoy** [reply]
Oh I didn't know that. Thanks for pointing that out @cderv
